### DISCUSSION TITLE
features request

### DISCUSSION THREAD

**Discussion by jekv2** [LawnchairLauncher/lawnchair#3213] [Ideas]
this is not my idea Nova launcher has it but when you long click the home icons there's pop up options for take you to Google Play store, export APK, could you guys implement this into open launcher?

![Screenshot_20221226-071426_Nova7](https://user-images.githubusercontent.com/7735249/209552894-77e2c905-a153-4077-8194-f205475f8015.jpg)

**Comment by jekv2** [comment]
I meant lawn chair I've been trying all kinds of launchers. got the two mixed up my bad

**Comment by VeryUsual** [comment]
Thanks for the suggestion! The idea of adding pop-up options for home icons is great and could definitely enhance Lawnchair. I might put together a pull request for this. If you have any more details or preferences, feel free to add them.
